packages/client/hmi-client/src/views/SimulationResult.vue

```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import { useRoute } from 'vue-router';
+import ResponsiveMatrix from '@/components/responsive-matrix/matrix.vue';
+
+const route = useRoute();
+
+watch(
+	() => route.params.simulationRunId,
+	async (id) => {
+		if (!id) return;
+
+		// FIXME: siwtch to different simulation run result
+		console.log('simulation run id changed to', id);
+	},
+	{ immediate: true }
+);
+
+const data = ref<any>([
+	[
+		{ S: 98, I: 2, R: 0 },
+		{ S: 50, I: 20, R: 30 },
+		{ S: 20, I: 30, R: 50 }
+	],
+	[
+		{ S: 98, I: 2, R: 0 },
+		{ S: 70, I: 10, R: 20 },
+		{ S: 50, I: 10, R: 40 }
+	],
+	[
+		{ S: 98, I: 2, R: 0 },
+		{ S: 20, I: 60, R: 20 },
+		{ S: 0, I: 30, R: 70 }
+	]
+]);
+
+const fillColorFn = (datum: any, _paramsMin: any, paramsMax: any): String => {
```

Comment:
No `any` please.

Reply:
if this needs to be typed we can use the following types.

```
import { CellData, ParamMinMax } from '@/types/ResponsiveMatrix';

const fillColorFn = (datum: CellData, _paramsMin: ParamMinMax, paramsMax: ParamMinMax): String => {
```